tests/FakeItEasy.Tests.Approval.netstd/ApiApproval.ApproveApiNetStd.approved.txt

```diff
@@ -436,7 +436,7 @@ namespace FakeItEasy.Creation
         FakeItEasy.Creation.IFakeOptions ConfigureFake(System.Action<object> action);
         FakeItEasy.Creation.IFakeOptions Implements(System.Type interfaceType);
         FakeItEasy.Creation.IFakeOptions Implements<TInterface>();
-        FakeItEasy.Creation.IFakeOptions WithAdditionalAttributes(System.Collections.Generic.IEnumerable<System.Reflection.Emit.CustomAttributeBuilder> customAttributeBuilders);
+        FakeItEasy.Creation.IFakeOptions WithAdditionalAttributes(params System.Linq.Expressions.Expression<>[] attributes);

```

Comment:
While we're here, and breaking the API, shall we remove the word "additional" and make these methods `WithAttributes`? I'm not seeing which other attributes these attributes can be said to be "additional" to.

All the supporting internal methods could be changed similarly.